modules/web/js/ballerina/utils/ace-worker.js

```diff
@@ -19,15 +19,20 @@
 import backends from 'bal_configs/backends';
 // This import defines ace/worker/mirror so we can ace.require ace/worker/mirror later
 import './ace-mirror-worker';
-ace.define('ace/worker/ballerina', ['require', 'exports', 'module'], (acequire, exports, module) => {
+
+const ace = global.ace;
+
+ace.define('ace/worker/ballerina', ['require', 'exports'], (acequire, exports) => {
```

Comment:
Yeah this is fine. I went through the ace documentation and this seems to be the way the ace object should be acquired